test/withStyles_test.jsx

```diff
@@ -1,555 +1,619 @@
+/* eslint-disable no-unused-vars */
+
```

Comment:
Why this tho?

Reply:
Without disabling this I can't do:
```
describe('foo', () => {
  let myUsedVar;
  
  beforeEach(() => {
    myUsedVar = imUsingIt();
  });
});
```

This fails. 😢

Reply:
Ah. Well that's a silly failure, but also Just Another Reason™ to [avoid setup](https://kentcdodds.com/blog/avoid-nesting-when-youre-testing) functions like `beforeEact` et al that share state, especially given that people can nest these types of things and make it super hard in a long test suite to figure out what is actually going on (I experienced this often on the Host product code with tests with nested blocks and setup helpers). I don't believe tests are were we should be keeping things DRY, and would much rather see each test verbosely express everything it needs.

Reply:
Good point. I'll definitely keep that in mind going forward ^_^